src/windows/views/tutorial.py

```diff
@@ -430,18 +430,21 @@ def __init__(self, win):
 
         # Configure tutorial frame
         self.dock.setTitleBarWidget(QWidget())  # Prevents window decoration
-        self.dock.setAttribute(Qt.WA_NoSystemBackground, True)
-        self.dock.setAttribute(Qt.WA_TranslucentBackground, True)
-        self.dock.setWindowFlags(Qt.FramelessWindowHint)
+        self.dock.setAttribute(Qt.WidgetAttribute.WA_NoSystemBackground, True)
+        self.dock.setAttribute(Qt.WidgetAttribute.WA_TranslucentBackground, True)
+        self.dock.setWindowFlags(Qt.WindowType.FramelessWindowHint)
         self.dock.setFloating(True)
 
         # Connect to interface dock widgets
-        self.win.dockFiles.visibilityChanged.connect(functools.partial(self.process, "dockFiles"))
-        self.win.dockTransitions.visibilityChanged.connect(functools.partial(self.process, "dockTransitions"))
-        self.win.dockEffects.visibilityChanged.connect(functools.partial(self.process, "dockEffects"))
-        self.win.dockProperties.visibilityChanged.connect(functools.partial(self.process, "dockProperties"))
-        self.win.dockVideo.visibilityChanged.connect(functools.partial(self.process, "dockVideo"))
-        self.win.dockEmojis.visibilityChanged.connect(functools.partial(self.process, "dockEmojis"))
+        for d in [
```

Comment:
This is actually more lines of code than he original, with the humble brag of not repeating any code, lol

Reply:
I ask you... which version is easier to glance at and understand?

Reply:
Fair question, but I'll throw one back at you: Which version will crash if `self.win` doesn't have a `dockProperties` attribute, or has one that's not a `QDockWidget` / lacks a `visibilityChanged` signal, and which version won't? The intent isn't merely to avoid cut-and-paste, it's to avoid assumptions about the structure of external classes.